CaloCluster/src/CaloClusterFast_module.cc

```diff
@@ -33,6 +33,7 @@ namespace mu2e {
             fhicl::Atom<double>         EnoiseCut         { Name("EnoiseCut"),         Comment("Minimum energy for a hit to be in a cluster") }; 
             fhicl::Atom<double>         ExpandCut         { Name("ExpandCut"),         Comment("Minimum energy for a hit to expand cluster") }; 
             fhicl::Atom<double>         deltaTime         { Name("deltaTime"),         Comment("Maximum time difference between seed and hit in cluster") }; 
+            fhicl::Atom<int>            minNSiPM          { Name("minNSiPM"),          Comment("Minimum number of SiPM contributing to the hit") }; 
```

Comment:
Do values other than 1 or 2 make sense for minNSipm?  If not, you might make this a bool (bothSipms) instead.

Reply:
TBH, I think that it's confusing to cut on that variable here. This module will receive the CaloHitCollection already processed and calibrated, so if there are crystals read out only by one SiPM the energy will be calibrated accordingly. @bechenard do you see a specific use case I'm missing?

Reply:
@Giani: This is to filter noise hits only reconstructed by a single SiPM. This is left as an option as I noticed this can improve the resolution a little bit, but more testing is needed to see if that will translate into a gain on the calo trigger. I'll follow Dave's suggestions and change the name to make this more clear.

Reply:
Actually, the situation is more complicated since one SiPM can be dead, in which case it should also be considered as "bothSiPMs". Since we will implement all these subtleties when we'll add DB support, I'll defer the work to that point and keep my lovely integer for now

Reply:
I still think that this selection should live in the module that creates the Hits. The clustering should be agnostic to the number of SiPM signals used to create a hit; flagging a hit a noisy is more complicated and I think we should not oversimplify it.